webapp/eslint.config.js

```diff
@@ -41,6 +42,7 @@ export default [
   },
   reactPlugin.configs.flat["jsx-runtime"],
   jsdocPlugin.configs["flat/recommended-typescript"],
+  cypressPlugin.configs.recommended,
```

Comment:
Keep only `cypressPlugin.configs.recommended`, the rest is already added: https://github.com/cypress-io/eslint-plugin-cypress/blob/master/lib/config/recommended.js

Reply:
I will check that